src/lib/ActionPrototype.js

```diff
@@ -1,29 +1,57 @@
-var fetcher = require('../lib/jsonFetcher');
-var AppDispatcher = require('../lib/ep-dispatcher');
-var KeyMirror = require('keymirror');
+import fetcher from '../lib/jsonFetcher'
+import AppDispatcher  from '../lib/ep-dispatcher'
```

Comment:
@BJK Do you want to leave this as `ep-dispatcher`? Seems like it will just cause unnecessary confusion since no one outside of Everplans will know what it refers to.